### PR TITLE
Ticket8527 ruff and py right corrections

### PR DESCRIPTION
**Pull request by davidkeymer**
Fixes to two files in the Config/Instrument Checker.

### CODE REVIEW

File: util/synoptic.py [hunk 1]
```diff
@@ -14,20 +14,20 @@ class SynopticUtils(object):
...
         return [
             f
             for f in CommonUtils.get_directory_contents_as_list(self.synoptics_path)
             if f.endswith(".xml")
         ]
 
-    def get_type_target_pairs(self, synoptic_xml):
+    def get_type_target_pairs(self, synoptic_xml: str) -> list[str]:
         """
         Returns a set of type, target pairs used in this synoptic
```
Comment (Tom-Willemsen) [inline util/synoptic.py:32]: Just purely from the docs this hint doesn't look quite right, the function says it returns `type, target pairs` so I'd expect something like `list[tuple[str, str]]` as the type hint?

**Review by Tom-Willemsen — COMMENTED** [review]
Looks like `ruff` wants you to reformat a couple of files

File: util/synoptic.py (hunk 1 above)
Comment (davidkeymer) [reply util/synoptic.py:32]: Thanks Tom  - should have spotted that!  Especially from line 50:
`result.append((type.text, target.text))`

File: util/gui.py [hunk 2]
```diff
@@ -11,43 +11,47 @@ class GuiUtils(object):
...
-    def get_gui_repo_at_release(self, version):
-        version = VersionUtils.extract_release_numbers_from_string(version)
-        branch_name = VersionUtils.convert_release_to_tag_name(*version)
+    def get_gui_repo_at_release(self, version_str: str) -> str | None:
+        version: list[int] = VersionUtils.extract_release_numbers_from_string(version_str)
+        branch_name: str = VersionUtils.convert_release_to_tag_name(*version)
         if not self.git.update_branch(branch_name, True):
             raise IOError(
                 "Couldn't check out GUI branch corresponding to release {}".format(version)
             )
```
Comment (Tom-Willemsen) [inline util/gui.py:24]: Not quite sure about this, the functions never returns so I think it's just `-> None`?

Comment (davidkeymer) [reply util/gui.py:24]: Quite right - missed that.

**Review by Tom-Willemsen — APPROVED** [review]

**Pull request merged**